packages/css/src/components/label/label.scss

```diff
@@ -15,4 +15,9 @@
 
   @include hyphenation;
   @include text-rendering;
+
+  & > .ams-hint {
```

Comment:
Can we generally apply these styles to `.ams-hint` without the parent selector?

Reply:
I'm making a dedicated Hint component that comes with its own styling.